chain-vote/src/gang/p256k1.rs

```diff
@@ -10,6 +11,10 @@ pub struct Scalar(IScalar);
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct GroupElement(Point);
 
+#[derive(Debug, thiserror::Error)]
+#[error("The length of the hash is incorrect. It should be at least BYTES_LEN of Coordinate.")]
```

Comment:
I think you can use BYTES_LEN in a format parameter expression here:
```suggestion
#[error("The length of the hash is incorrect. It should be at least {} bytes", Coordinate::BYTES_LEN)]
```